src/Tmds.Ssh/ISshClientImplementation.cs

```diff
@@ -11,6 +11,7 @@ interface ISshClientImplementation
 {
     Task ConnectAsync(CancellationToken cancellationToken);
     Task<ISshChannel> OpenRemoteProcessChannelAsync(Type channelType, string command, CancellationToken cancellationToken);
+    Task<ISshChannel> OpenRemoteSubsystemChannelAsync(Type channelType, string subsystem, CancellationToken cancellationToken);
```

Comment:
I'm unsure how you want to handle this, IIRC the default access modifier for an interface is `public` so this may be a breaking change API wise.

Reply:
This interface itself is not public so there is no backwards compatibility concern.

In addition to the managed implementation, there once also was a [libssh](https://www.libssh.org) based backend. This interface is a reminiscent of that, and something to be removed.

fyi, C# has a feature to extend interfaces so it doesn't affect existing implementations, see [default interface methods](https://learn.microsoft.com/en-us/dotnet/csharp/advanced-topics/interface-implementation/default-interface-methods-versions).

Reply:
Thanks for clarifying